$NetBSD: patch-libs_alignment_CMakeLists.txt,v 1.2 2024/08/05 14:51:37 adam Exp $

On Darwin, set correct install name dir for plug-ins.

--- libs/alignment/CMakeLists.txt.orig	2024-08-05 10:54:29.125435547 +0000
+++ libs/alignment/CMakeLists.txt
@@ -145,6 +145,9 @@ set(NearestMathPlugin_SRCS
 )
 
 add_library(indi_Nearest_MathPlugin SHARED ${NearestMathPlugin_SRCS})
+IF(APPLE)
+    set_property(TARGET indi_Nearest_MathPlugin PROPERTY INSTALL_NAME_DIR ${INDI_MATH_PLUGINS_DIRECTORY})
+ENDIF()
 target_include_directories(indi_Nearest_MathPlugin
     PRIVATE
         ${libindi_SOURCE_DIR}/libs/indicore
@@ -168,6 +171,9 @@ set(SVDMathPlugin_SRCS
 
 add_library(indi_SVD_MathPlugin SHARED ${SVDMathPlugin_SRCS})
 set_property(TARGET indi_SVD_MathPlugin APPEND PROPERTY COMPILE_DEFINITIONS SVD_TRANSFORM_MATRIX)
+IF(APPLE)
+    set_property(TARGET indi_SVD_MathPlugin PROPERTY INSTALL_NAME_DIR ${INDI_MATH_PLUGINS_DIRECTORY})
+ENDIF()
 target_link_libraries(indi_SVD_MathPlugin
     indidriver
     $<$<PLATFORM_ID:CYGWIN>:AlignmentDriver>
